docs/code_push/_category_.json

```diff
@@ -0,0 +1,10 @@
+{
+    "label": "☁️ Code Push",
+    "position": 3,
+    "link": {
+      "title": "Code Push",
+      "slug": "code-push",
+      "type": "generated-index",
+      "description": "Learn everything you need to add over the air updates (code push) to your Flutter application."
```

Comment:
I'd probably reverse the parenthetical "to add code push (over the air updates)" 🤷   The point of using "code push" is mostly to piggy back on the naming used by RN.  e.g. https://microsoft.github.io/code-push/ which might be a term users have heard. 🤷